metron-deployment/roles/bro/tasks/metron-bro-plugin-kafka.yml

```diff
@@ -15,36 +15,34 @@
 #  limitations under the License.
 #
 ---
-- name: Clone metron-bro-plugin-kafka into /tmp
-  git:
-    repo: https://github.com/apache/metron-bro-plugin-kafka
-    version: master
-    dest: /tmp/metron-bro-plugin-kafka
+- name: Install bro-pkg
+  environment:
+    LD_LIBRARY_PATH: "{{ python27_lib }}"
+  command: "{{ python27_bin }}/pip install bro-pkg"
+
+- name: Configure bro-pkg
+  environment:
+    PATH: "{{ git29_bin }}:{{ bro_bin }}:{{ ansible_env.PATH }}"
+    LD_LIBRARY_PATH: "{{ python27_lib }}"
+  command: "{{ python27_bin }}/bro-pkg autoconfig"
 
-- name: Compile and install the plugin
-  shell: "{{ item }}"
+- name: Install the metron-bro-plugin-kafka package
   environment:
+    PATH: "{{ git29_bin }}:{{ bro_bin }}:{{ ansible_env.PATH }}"
+    LD_LIBRARY_PATH: "{{ httpd24_lib }}:{{ python27_lib }}"
     CXX: /opt/rh/devtoolset-4/root/usr/bin/g++
     CC: /opt/rh/devtoolset-4/root/usr/bin/gcc
-  args:
-    chdir: "/tmp/metron-bro-plugin-kafka"
-    creates: "{{ bro_home }}/lib/bro/plugins/BRO_KAFKA"
-  with_items:
-    - rm -rf build/
-    - "./configure --bro-dist=/tmp/bro-{{ bro_version }} --install-root={{ bro_home }}/lib/bro/plugins/ --with-librdkafka={{ librdkafka_home }}"
-    - make
-    - make install
+  command: "{{ python27_bin }}/bro-pkg install https://github.com/jonzeolla/metron-bro-plugin-kafka --force --version {{ metron_bro_plugin_kafka_version }}"
```

Comment:
See my Contributor Comments for why this is `https://github.com/jonzeolla/metron-bro-plugin-kafka` instead of `apache/metron-bro-plugin-kafka`.  The reason why it won't need to be a full URL when it moves to `apache/` is because of [this](https://github.com/bro/packages/blob/master/apache/bro-pkg.index).